frisc/src/cpu.fil

```diff
@@ -0,0 +1,30 @@
+import "frisc/src/alu.fil";
+import "frisc/src/decode.fil";
+import "frisc/src/regfile.fil";
+import "frisc/src/extras/extras.fil";
+
+comp cpu<G:1> (
```

Comment:
One random, completely undocumented, naming scheme I abide by is CamelCasing component names so that they look distinct from ports and invocations. Also, somehow, they feel similar to classes since we have to instantiate them using `new`.

Reply:
I like this, and also realized I was not deliberate with my naming scheme here at all so I'll change it to this